### ISSUE TITLE
Replace admin program-editing modals with USWDS modal component

### ISSUE THREAD

**Issue by melanie-exygy** [civiform/civiform#6260]
#### Describe the solution you'd like.

As part of our effort to migrate to the USWDS modal component, we'd like to replace the following admin program-editing modals with the [USWDS component](https://designsystem.digital.gov/components/modal/).

1. Publishing all draft programs ("Do you want to publish all draft programs?")
2. Publishing a single draft program ("Are you sure you want to publish...")
3. Confirm pre-screener change

For a screenshot and description of these modals, see this [document](https://docs.google.com/document/d/1b1SMIoDCw_pAq7w0QN9B3hQyGB0wV-_QXkGASqyfqj8/edit?usp=sharing)

#### Additional Context

We can utilize this [helper function](https://github.com/civiform/civiform/blob/88813618cdc44338c35986a11b34c387c6f60230/server/app/views/ViewUtils.java#L394), modifying it as needed or putting it into its own utility class.

#### Done when:

All 3 of these modals use the USWDS component.

**Comment by rockycodes** [comment]
#2 will be done when we turn on universal questions: https://github.com/civiform/civiform/pull/6116

**Comment by melanie-exygy** [comment]
> #2 will be turned into a modal rather than a dialogue when we turn on universal questions: #6116

Great to know.  Thanks @rockycodes!  It'll be a modal, but not a USWDS modal, right?

**Comment by rockycodes** [comment]
> > #2 will be turned into a modal rather than a dialogue when we turn on universal questions: #6116
> 
> Great to know. Thanks @rockycodes! It'll be a modal, but not a USWDS modal, right?

Yep! Good point, it will still need to be refactored to be a USWDS modal.
